src/Commands/IntakingCondition.cpp

```diff
@@ -15,7 +15,7 @@
 #include "Subsystems/Arm.h"
 #include "SetSlingDisengagement.h"
 
-IntakingCondition::IntakingCondition() {
+IntakingCondition::IntakingCondition(int position) {
```

Comment:
This is an old IntakingCondition. This command should not be run anywhere.

No change required.